app/views/spree/orders/_totals_footer.html.haml

```diff
@@ -23,6 +23,22 @@
           = t :order_total_price
       %td.text-right.total
         %h5#order_total= order.display_total.to_html
+    %tr.total
+      %td.text-right{colspan: "3"}
+        %h5
+          = t :order_amount_paid
+      %td.text-right.total
+        %h5
+          = order.payment_total
```

Comment:
Also, I notice that all other figures are formatted with a currency symbol. Although `payment_total` isn't an instance of Spree::Money like the others, we can wrap it like so:

```suggestion
          = Spree::Money(order.payment_total).to_html
```

Reply:
Added display_payment_total method to Orders model.